keyboards/id80/readme.md

```diff
@@ -12,4 +12,8 @@ Make example for this keyboard (after setting up your build environment):
 
     make id80:default
 
+or for (CH-based) ISO layout:
+    
+    make id80:iso_ch
```

Comment:
```suggestion
Flashing example for this keyboard:

    make id80:default:flash
```

I don't think it's necessary to add an example for the ISO layout, as the process is fundamentally the same.
But if you are already polishing this board's support, I'd like to ask you to add the flashing example and clear instructions on how to reset the board into bootloader mode.